src/preset_cli/cli/superset/sync/dbt/command.py

```diff
@@ -111,7 +111,7 @@
 )
 @raise_cli_errors
 @click.pass_context
-def dbt_core(  # pylint: disable=too-many-arguments, too-many-branches, too-many-locals ,too-many-statements
+def dbt_core(  # pylint: disable=too-many-arguments, too-many-branches, too-many-locals ,too-many-statements # noqa: C901
```

Comment:
There's some logic getting deprecated in this method that should allow us to remove `# noqa: C901` soon. I could also plan a bigger refactor to break this down into smaller methods.